README.md

```diff
@@ -65,3 +65,16 @@ Use `py.test` to execute the unit tests.
 # License
 
 GPLv3. See the [license](license.txt) file.
+
+# Network Protocol
```

Comment:
So regarding detailing the network protocol in `README.md`, I think it's better left to a seperate document (Perhaps the wiki?)